constants/paths.js

```diff
@@ -1,15 +1,32 @@
 const path = require("path");
 
+const TEMP_DIR = path.join(__dirname, "../temp");
+
 const TEMP_DIR_VIDEOS = {
-  MAIN: path.join(__dirname, "../temp/videos/main/"),
+  VIDEOS: path.join(__dirname, "../temp/videos"),
```

Comment:
VIDEOS path를 사용하려면 TEMP_DIR_VIDEOS.VIDEOS가 되어야하는데 조금 어색한 것 같아서 변수명을 변경하는 것은 어떨까요? 
예를 들어,
```suggestion
  FOLDER: path.join(__dirname, "../temp/videos"),
```

Reply:
중복된 명명이라 그렇겠네요. 지금 당장 필요하지 않아도 혹시몰라 넣어둔 경로라 보기 좋은 상태로 변경해두겠습니다!

Reply:
중복된 명명이라 이게 더 좋은 것 같네요! 변경해두겠습니다.